### PR TITLE
several small cleanups

### PR DESCRIPTION
**Pull request by phritz**
separate logical commits

### CODE REVIEW

File: cmd/repl/main.go [hunk 1]
```diff
@@ -319,7 +319,7 @@ func drop(parent *kingpin.Application, gsp gsp, in io.Reader, out io.Writer) {
 }
 
 func logCmd(parent *kingpin.Application, gdb gdb, out io.Writer) {
-	kc := parent.Command("log", "Displays the history of a this client database.")
+	kc := parent.Command("log", "Displays the history of the database up to but not including the latest Snapshot.")
```
Comment (aboodman) [inline cmd/repl/main.go:322]: [help] I don't understand what this means. We don't display any of the pending commits or their basis?

Comment (phritz) [reply cmd/repl/main.go:322]: the previous implementation showed all commits from head back to to but not including the latest genesis/snapshot. this change does not change that behavior. this change makes the description more accurate. 

if you are thinking that the behavior of log could be more useful, i agree.

Comment (aboodman) [reply cmd/repl/main.go:322]: I see - I parsed "up" differently than you meant it. I thought the comment was saying it showed history from beginning of time to the latest snapshot. You are saying it shows history latest snapshot to now.

Comment (phritz) [reply cmd/repl/main.go:322]: yep

**Pull request merged**